gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
This gets into the two scenarios outlined at https://github.com/Polymer/docs/pull/1641#issuecomment-224081435

In scenario 2), the App Shell (`index.html`, for this site) is the only thing that needs to be kept up to date, and it then pulls in dynamic content at runtime and populates the shell with the body of the specific page that's actually requested. That implies that everything else on the page stays the same (header/side nav/etc.) and just the dynamic content gets swapped in/out depending on the URL.

If we're in some sort of hybrid of the two scenarios, then things can be messy. Should we all discuss this off thread to avoid the back and forth and get some certainty?